### PR TITLE
Feature/df 1567 ch fiche setup

### PR DESCRIPTION
**Pull request by emina-str**

### CODE REVIEW

File: domain/src/test/java/ch/admin/seco/alv/service/competencecatalog/domain/chfiche/ChFicheIdFixture.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
+package ch.admin.seco.alv.service.competencecatalog.domain.chfiche;
+
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Set;
+
+public enum ChFicheIdFixture {
+
+	CHF1, CHF2, CHF3, CHF4, CHF5,
```
Comment (cjpsycos) [inline domain/src/test/java/ch/admin/seco/alv/service/competencecatalog/domain/chfiche/ChFicheIdFixture.java]: I think five are enough ;)

File: application/src/test/java/ch/admin/seco/alv/service/competencecatalog/application/chfiche/ChFicheApplicationServiceTest.java [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+				chFiche(CHF1.id(), String.format("%s-%s", "Title", CHF1.id().getValue())),
+				chFiche(CHF2.id(), String.format("%s-%s", "Title", CHF2.id().getValue())),
+				chFiche(CHF3.id(), String.format("%s-%s", "Title", CHF3.id().getValue())),
+				chFiche(CHF4.id(), String.format("%s-%s", "Title", CHF4.id().getValue())),
+				chFiche(CHF5.id(), String.format("%s-%s", "Title", CHF5.id().getValue())),
+				chFiche(CHF6.id(), String.format("%s-%s", "Title", CHF6.id().getValue())),
+				chFiche(CHF7.id(), String.format("%s-%s", "Title", CHF7.id().getValue())),
+				chFiche(CHF8.id(), String.format("%s-%s", "Title", CHF8.id().getValue())),
+				chFiche(CHF9.id(), String.format("%s-%s", "Title", CHF9.id().getValue())),
+				chFiche(CHF10.id(), String.format("%s-%s", "Title", CHF10.id().getValue()))
```
Comment (cjpsycos) [inline application/src/test/java/ch/admin/seco/alv/service/competencecatalog/application/chfiche/ChFicheApplicationServiceTest.java]: In this test, the title is not relevant. on the fixture I would make a chFiche() only with id and put "lorem ipsum" as title

File: domain/src/test/java/ch/admin/seco/alv/service/competencecatalog/domain/chfiche/ChFicheFixture.java [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+import static ch.admin.seco.alv.service.competencecatalog.domain.chfiche.CompetenceFixture.competence;
+import static ch.admin.seco.alv.service.competencecatalog.domain.occupation.BfsCodeFixture.BFS1;
+
+public class ChFicheFixture {
+
+	private ChFicheFixture() {
+	}
+
+	public static ChFiche chFiche(ChFicheId id, String title) {
+		return chFiche(id, title, "");
```
Comment (cjpsycos) [inline domain/src/test/java/ch/admin/seco/alv/service/competencecatalog/domain/chfiche/ChFicheFixture.java]: description could be null

File: domain/src/test/java/ch/admin/seco/alv/service/competencecatalog/domain/chfiche/ChFicheIdFixture.java (hunk 1 above)
Comment (emina-str) [reply domain/src/test/java/ch/admin/seco/alv/service/competencecatalog/domain/chfiche/ChFicheIdFixture.java]: Ok :)

File: application/src/test/java/ch/admin/seco/alv/service/competencecatalog/application/chfiche/ChFicheApplicationServiceTest.java (hunk 2 above)
Comment (emina-str) [reply application/src/test/java/ch/admin/seco/alv/service/competencecatalog/application/chfiche/ChFicheApplicationServiceTest.java]: Fixed, the fixture and the test.

File: domain/src/test/java/ch/admin/seco/alv/service/competencecatalog/domain/chfiche/ChFicheFixture.java (hunk 3 above)
Comment (emina-str) [reply domain/src/test/java/ch/admin/seco/alv/service/competencecatalog/domain/chfiche/ChFicheFixture.java]: fixed.

File: application/src/main/java/ch/admin/seco/alv/service/competencecatalog/application/chfiche/ChFicheApplicationService.java [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.PageImpl;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.stereotype.Service;
+
+import static ch.admin.seco.alv.service.competencecatalog.application.chfiche.ChFicheAssembler.*;
+import static ch.admin.seco.alv.service.competencecatalog.core.events.DomainEventPublisher.publish;
+
+@Service
+public class ChFicheApplicationService {
```
Comment (fabiangotzen) [inline application/src/main/java/ch/admin/seco/alv/service/competencecatalog/application/chfiche/ChFicheApplicationService.java]: Should probably be @Transactional

**Pull request merged**